lerna.json

```diff
@@ -1,7 +1,7 @@
 {
   "lerna": "2.5.1",
```

Comment:
Installed lerna version is much newer. Not use if it matters to update the version here as well.

It's not strictly related to the changeset of this PR and it was like that before as well, but I noticed it now. Might be a good time to check it